vmm/src/lib.rs

```diff
@@ -1078,6 +1080,9 @@ impl Vmm {
                 )));
             }
             info!("Migration complete");
+            // Disable dirty logging
+            #[cfg(all(feature = "mshv", target_arch = "x86_64"))]
+            vm.complete_live_migration()?;
```

Comment:
I think it is better to rename this function to `vm.complete_memory_dirty_log` and call it before `Request::complete().write_to(&mut socket)?;`.